Kickstarter-iOS/Features/ThanksProjects/Datasource/ThanksProjectsDataSource.swift

```diff
@@ -4,17 +4,12 @@ import UIKit
 
 internal final class ThanksProjectsDataSource: ValueCellDataSource {
   internal func loadData(projects: [Project],
-                         category: KsApi.Category,
-                         nativeProjectCardsVariant: OptimizelyExperiment.Variant = .control) {
+                         category: KsApi.Category) {
     let values = projects.map { (project) -> DiscoveryProjectCellRowValue in
       DiscoveryProjectCellRowValue(project: project, category: category, params: nil)
     }
 
-    if nativeProjectCardsVariant == .variant1 {
-      self.set(values: values, cellClass: DiscoveryProjectCardCell.self, inSection: 0)
```

Comment:
so you're suggesting that we remove `DiscoveryProjectsDataSource` and `ThanksProjectsDataSource`?

Reply:
I think its being used when loadData is called in `ThanksProjectsDataSource`

Reply:
yes it does! I was just thinking that because we removed the `set` function in `DiscoveryProjectsDataSource` and `ThanksProjectsDataSource` for this cell we didn't need to have it around anymore. Up to you - I'll approve without this change.